__tests__/session.spec.js

```diff
@@ -1,19 +1,20 @@
 /*
  * Test session mechanism
  */
-const seed = require('./seed');
-const Session = require("../server/models/Session");
 const jestExpect = require("expect");
 const sinon = require("sinon");
 const chai = require("chai");
 const sinonChai = require("sinon-chai");
+const Session = require("../server/models/Session");
+const seed = require('./seed');
+
 chai.use(sinonChai);
 const {expect} = chai;
 
 
 describe("Session integration", () => {
   beforeEach(async () => {
-    //before all, seed data to DB
+    // before all, seed data to DB
```

Comment:
We also used prettier to format the code?

Reply:
yes, yarn lint uses prettier for cosmetics like indentation, quotes etc and eslint for the rest

Reply:
Can NPM make it too? Or all we need to switch to yarn? I never used yarn, how about it?

Reply:
oh, right , it doesn't matter much in this case, I used yarn as a script runner, but it can be npm instead. Eventually it calls whatever is specified in package.json under "lint" command